### PR TITLE
Quartermaster milestone 2 report

### PR DESCRIPTION
**Pull request by mirkoboehm**
Needs review.

### CODE REVIEW

File: content/blog/qmstr-milestone-2-report.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
++++
+title = "Quartermaster Milestone 2 Development Report: Python client modules, SPDX, more automation"
+date = "2018-08-08T00:00:00+01:00"
+tags = ["qmstr", "FOSS", "compliance", "milestone", "report"]
+categories = ["qmstr"]
+banner = "img/blog/qmstr-ms-2-report-title.jpg"
++++
+
+After another quarter of intense software development, We are proud to
```
Comment (GiasemiSh) [inline content/blog/qmstr-milestone-2-report.md]: We <- lowercase

File: content/blog/qmstr-milestone-2-report.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+announce the availablility of Quartermaster v0.2. Quartermaster is a
```
Comment (GiasemiSh) [inline content/blog/qmstr-milestone-2-report.md]: "availability" spelling mistake.

File: content/blog/qmstr-milestone-2-report.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+Open Source software and developed under a collaborative open
+governance model. Get the [source code from Github while it is
+hot](https://github.com/QMSTR/qmstr/releases/tag/v0.2)! Read more for
+all the details on the new release.
+<!--more-->
+
+# Quartermaster Version 0.2
+
+As announced in an earlier blog post, a milestone version for
+Quartermaster will be tagged roughly evey 3 months. Quartermaster as a
```
Comment (hemarkus) [inline content/blog/qmstr-milestone-2-report.md]: every 3 month

File: content/blog/qmstr-milestone-2-report.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+The `qmstrctl` tool has been refactored from what was previously
+known as ~~`qmstr-cli`~~ and provides functions for the client side to
+control and interact with the master. In fact, as the tutorial is
+showing, it now starts and quits the master as well. Other functions
+include triggering the analysis and reporting phases, querying the
+master Log output and copying the output data to the client side. In
+the process, the ports and container names have been abstracted so
+that multiple masters running on the same host do not interfere with
+eah other. All tools now use a consistent pflag-based command line
```
Comment (hemarkus) [inline content/blog/qmstr-milestone-2-report.md]: each

**Review by hemarkus — CHANGES_REQUESTED** [review]
few typos to fix

File: content/blog/qmstr-milestone-2-report.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+are now becoming a reality. A [new
+tutorial](https://github.com/QMSTR/qmstr/blob/master/doc/Getting-Started.md)
+illustrates the philosophy and basic use of the Quartermaster
+toolchain. The build system has been updated to simplify the
+installation of the client and master side tools. An updated client
+side control tool (`qmstrctl`) is used to
+interact with the master. Based on the functionality provided with
+this release, the team will focus on implementing a number of
+real-life use cases that will be published on the project web site and
+serves as references on how to use the toolchain.
```
Comment (lisa-noeth) [inline content/blog/qmstr-milestone-2-report.md]: serve (because plural)

File: content/blog/qmstr-milestone-2-report.md (hunk 4 above)
Comment (lisa-noeth) [inline content/blog/qmstr-milestone-2-report.md]: flag-based?

Comment (hemarkus) [reply content/blog/qmstr-milestone-2-report.md]: POSIX/GNU-style command line flags

**Comment by lisa-noeth** [comment]
I found a few more typos, other than that, it looks fantastic. Great stuff.

**Pull request merged**